### ISSUE TITLE
export HTML schedule on schedule release & intervals

### ISSUE THREAD

**Issue by luto** [pretalx/pretalx#277]
Static schedule exports can currently only be triggered manually via a management command. But we know when the data changes so looking at #275 we should do that automatically, if the user wishes us to do so.

- [x] generate export on schedule release #297
- [ ] maybe generate in regular intervals
- [ ] maybe generate on talk changes, e.g. the title changes: is there a way to catch bluk updates?

**Comment by rixx** [comment]
There is no good way to catch bulk updates, but we don't really do those. Instead on updating on talk changes directly, how about storing a dirty flag that will be reset once the schedule has been regenerated?

**Comment by rixx** [comment]
The HTML export also needs to be regenerated when the organiser uploads a changed CSS or logo.

**Comment by rixx** [comment]
We now have a good way to do this!

```python
event.cache.set('rebuild_schedule_export', True, None)
```

will have to be added when

- [ ] A talk is confirmed
- [ ] A talk is unconfirmed
- [ ] A talk changes in a visible way
- [ ] A speaker changes in a visible way
- [ ] Related data to a talk or a speaker (question answers!)

**Comment by rixx** [comment]
I decided that it doesn't hurt to be careful, so this flag is set on any submission save and speaker save, currently (as long as the schedule is configured to be exported on release). I think it would be way harder to make sure a change does/does not affect the schedule than to just trigger a regeneration of the schedule export periodically. I've limited the automatic regeneration to 30 minutes to be on the safe side in terms of DoS.
